nautobot/utilities/forms/forms.py

```diff
@@ -279,7 +279,9 @@ def __init__(self, *args, filterset_class=None, **kwargs):
         model = self.filterset_class._meta.model
 
         if self.filterset_class is not None:
-            filterset_class = self.filterset_class()
+            filterset_class = self.filterset_class
```

Comment:
From an API standpoint, I don't like passing an *instance* to a param specifically named `filterset_class`. Can we maybe augment the params to include a `filterset` param as a separate option instead?